docs/testing.rst

```diff
@@ -4,118 +4,76 @@
 Testing
```

Comment:
The info here seems to be duplicated from contributing.rst do we really need it in two places?

Reply:
Yep you are correct. That was one of the files on the list to remove. I've merged any recent changes into `contributing.rst` and removed `testing.rst` now.